### PR TITLE
Sanity Tests - Refacto all sanity tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto all sanity tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26217)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

**Review by boubkerbribri — COMMENTED** [review]
Could you please check problem with step identifiers

File: tests/UI/campaigns/sanity/02_ordersBO/01_filterOrders.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_ordersBO_filterOrders';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/02_ordersBO/01_filterOrders.js]:
```suggestion
// Import pages
```

File: tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js [hunk 2]
```diff
@@ -1,18 +1,24 @@
 require('module-alias/register');
 // Using chai
 const {expect} = require('chai');
+
+// import utils
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/02_ordersBO/02_editOrder.js]:
```suggestion
// Import utils
```

File: tests/UI/campaigns/sanity/03_productsBO/01_filterProducts.js [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_productsBO_filterProducts';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/03_productsBO/01_filterProducts.js]:
```suggestion
// Import pages
```

File: tests/UI/campaigns/sanity/03_productsBO/02_CRUDStandardProductInBO.js [hunk 4]
```diff
@@ -1,35 +1,46 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_productsBO_CRUDStandardProductInBO';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import BO pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/03_productsBO/02_CRUDStandardProductInBO.js]:
```suggestion
// Import BO pages
```

File: tests/UI/campaigns/sanity/03_productsBO/03_CRUDStandardProductWithCombinationsInBO.js [hunk 5]
```diff
@@ -1,33 +1,44 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_productsBO_CRUDStandardProductWithCombinationsInBO';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import BO pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/03_productsBO/03_CRUDStandardProductWithCombinationsInBO.js]:
```suggestion
// Import BO pages
```

File: tests/UI/campaigns/sanity/03_productsBO/04_deleteProduct.js [hunk 6]
```diff
@@ -1,19 +1,24 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_productsBO_deleteProduct';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/03_productsBO/04_deleteProduct.js]:
```suggestion
// Import pages
```

File: tests/UI/campaigns/sanity/03_productsBO/05_deleteProductsWithBulkActions.js [hunk 7]
```diff
@@ -1,19 +1,24 @@
...
 const helper = require('@utils/helpers');
-const loginCommon = require('@commonTests/loginBO');
 const testContext = require('@utils/testContext');
 
-const baseContext = 'sanity_productsBO_deleteProductsWithBulkActions';
+// Import login steps
+const loginCommon = require('@commonTests/loginBO');
 
-// importing pages
+// import pages
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/03_productsBO/05_deleteProductsWithBulkActions.js]:
```suggestion
// Import pages
```

**Review by Progi1984 — COMMENTED** [review]
@nesrineabdmouleh Thank you for your contribution. I have brought you some comments about typos.

File: tests/UI/campaigns/sanity/02_ordersBO/01_filterOrders.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/sanity/02_ordersBO/01_filterOrders.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**